sdk/search/search-documents/CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Release History
 
+## 11.1.0 (2021-02-11)
+
+### Changes since 11.0.3
+
+- The `@azure/search-documents` has been moved to V6 SDK Generator. The following types `BlobIndexerDataToExtract`, `BlobIndexerImageAction`, `BlobIndexerParsingMode`, `BlobIndexerPDFTextRotationAlgorithm` has been changed to `string`. Please refer [#12829](https://github.com/Azure/azure-sdk-for-js/pull/12829) for further details.
+- Added Batching ability to the search SDK. The `SearchIndexingBufferedSender` class enables the user to perform indexing documents in batch mode. There are several user configurable properties such as `autoFlush`, `flushWindowInMs`, `throttlingDelayInMs`, etc.
```

Comment:
In terms of relevance, the batching ability would be of more value to the user than the types changing to string. So, I would pull that up. Regarding the type change, it would be good to share why this happened. And the answer cannot be "moved to v6 SDK generator", because that does not make much sense to the user in itself. Something more in terms of extensible enums would make more sense. How about the below?

```suggestion
- Added Batching ability to the search SDK. The `SearchIndexingBufferedSender` class enables the user to perform indexing documents in batch mode. There are several user configurable properties such as `autoFlush`, `flushWindowInMs`, `throttlingDelayInMs`, etc.
- The types `BlobIndexerDataToExtract`, `BlobIndexerImageAction`, `BlobIndexerParsingMode`, `BlobIndexerPDFTextRotationAlgorithm` have changed from union of string literals to string. This is to support the service definition for these types to be extensible enum. The documentation on methods that use these have been updated with known values that can be used for these types.
```

Reply:
Feel free to merge this PR and do the changelog update in another PR